src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -360,6 +363,14 @@ public void Dispose()
                 }
             }
 
+            public void Cancel()
```

Comment:
How does the Http2Stream get removed from the dictionary in this case?

Reply:
It currently isn't removed here, but it should be. I'll make that change.